doc/languages-frameworks/python.section.md

```diff
@@ -206,15 +206,15 @@ build inputs (see "Specifying dependencies"). The following are of special
 interest for Python packages, either because these are primarily used, or
 because their behaviour is different:
 
-* `nativeBuildInputs ? []`: Build-time only dependencies. Typically executables
-  as well as the items listed in `setup_requires`.
+* `nativeBuildInputs ? []`: Build-time only dependencies. Typically executables.
+* `build-system ? []`: Build-time only Python dependencies. Items listed in `build-system.requires`/`setup_requires`.
 * `buildInputs ? []`: Build and/or run-time dependencies that need to be
   compiled for the host machine. Typically non-Python libraries which are being
   linked.
 * `nativeCheckInputs ? []`: Dependencies needed for running the [`checkPhase`](#ssec-check-phase). These
   are added to [`nativeBuildInputs`](#var-stdenv-nativeBuildInputs) when [`doCheck = true`](#var-stdenv-doCheck). Items listed in
   `tests_require` go here.
-* `propagatedBuildInputs ? []`: Aside from propagating dependencies,
+* `dependencies ? []`: Aside from propagating dependencies,
```

Comment:
```suggestion
* `propagatedBuildInputs ? []`: Aside from propagating dependencies,
* `dependencies ? []`: Run-time Python dependencies. Items listed in `project.requires`/`python_requires`.
```
`propagatedBuildInputs` will be much less frequent, but may be necessary in some cases.

Reply:
Yes, but that's an `stdenv.mkDerivation` argument, not really a `buildPythonPackage` one.